src/reducers/authSlice.js

```diff
@@ -0,0 +1,70 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+const axios = require("axios");
+
+export const login = createAsyncThunk(
+  'auth/login', 
+  async (loginData, { rejectWithValue }) => {
+  const response = await axios
+    .post(
+      "http://localhost:8000/graphql", //update
```

Comment:
How about providing this from react env?
If we will have to change the URL, we'll have to make changes at multiple places unnecessarily.

Reply:
Yeah! This has to be updated. Idk why but .env variables aren't working right now. Added //update comment anywhere it needs to be done; will be fixed afterwards

Reply:
What's happening with .env?
Keep in mind that we need to have the prefix `REACT_APP_` in any variable to be able to use in react!

Reply:
This was a life saviour! Thanks